### PR TITLE
Feature support welly

### PR DESCRIPTION
**Pull request by neyberson**
Add welly to pozo

### CODE REVIEW

File: pozo/graphs.py [hunk 1]
```diff
@@ -122,6 +131,52 @@ def add_las_object(self, ar, **kwargs):
...
+    def add_welly_object(self, ar, **kwargs):
+        include = kwargs.get('include', [])
+        exclude = kwargs.get('exclude', [])
+        yaxis = kwargs.get('yaxis', None)
+        yaxis_name = kwargs.get('yaxis_name',"DEPTH")
+        yaxis_unit = kwargs.get('yaxis_unit', None)
+        
+        warnings.filterwarnings("ignore", message="Index.is_numeric is deprecated.")
+        for curve in ar.data.values():
+            print(f"{curve.mnemonic} w/ units: {curve.units} w/ shape {curve.values.shape}")
```
Comment (ayjayt) [inline pozo/graphs.py:144]: ya no

File: pozo/graphs.py [hunk 2]
```diff
@@ -122,6 +131,52 @@ def add_las_object(self, ar, **kwargs):
...
+            
+        if yaxis is not None:
+            yaxis_name = None
+            if hasattr(yaxis, "units"):
+                yaxis_unit = yaxis.units
+            else:
+                warnings.warn("Not sure what yaxis units are.") # TODO
+        elif yaxis_name in ar.data.keys():
+            yaxis = ar.data[yaxis_name]
+            yaxis_unit = pzu.parse_unit_from_curve(ar.data[yaxis_name])
```
Comment (ayjayt) [inline pozo/graphs.py:154]: debe ser igual a linea 168?

**Review by ayjayt — CHANGES_REQUESTED** [review]
casi listo

File: pozo/graphs.py (hunk 1 above)
Comment (neyberson) [reply pozo/graphs.py:144]: Are you talking about the filterwarnings?

File: pozo/graphs.py (hunk 2 above)
Comment (neyberson) [reply pozo/graphs.py:154]: Done

File: pozo/graphs.py (hunk 1 above)
Comment (neyberson) [reply pozo/graphs.py]: Done

**Comment by ayjayt** [comment]
Thanks @neyberson, your branch has been merged and I'm about to launch a new version. You can delete your branch if you'd like, and update your main if you're using a local repository with `pip -e`.

**Pull request merged**